### PR TITLE
Added Cloudmersive PDF connector

### PR DESCRIPTION
**Pull request by Cloudmersive**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ X ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### CODE REVIEW

File: connectors/Cloudmersive PDF/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1 @@
...ytes":{"format":"byte","description":"Optional: Bytes of the input file to operate on","type":"string"},"InputFileUrl":{"description":"Optional: URL of a file to operate on as input.  This can be a public URL, or you can also use the begin-editing API to upload a document and pass in the secure URL result from that operation as the URL here (this URL is not public).","type":"string"},"StartDeletePageNumber":{"format":"int32","description":"Page number (1-based) to start deleting pages; inclusive","type":"integer"},"EndDeletePageNumber":{"format":"int32","description":"Page number (1-based) to stop deleting pages; inclusive","type":"integer"}}},"GetDocxHeadersAndFootersRequest":{"description":"Input to a Get Tables request","type":"object","properties":{"InputFileBytes":{"format":"byte","description":"Optional: Bytes of the input file to operate on","type":"string"},"InputFileUrl":{"description":"Optional: URL of a file to operate on as input.  This can be a public URL, or you can also use the begin-editing API to upload a document and pass in the secure URL result from that operation as the URL here (this URL is not public).","type":"string"}}},"GetDocxHeadersAndFootersResponse":{"
```
Comment (v-sriyen) [inline connectors/Cloudmersive PDF/apiDefinition.swagger.json]: @Cloudmersive Could you please fix the indentations as the entire swagger is showing in one line.

File: connectors/Cloudmersive PDF/readme.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+- [Register](https://account.cloudmersive.com/signup) for a Cloudmersive Account
+- [Sign In](https://account.cloudmersive.com/login) with your Cloudmersive Account and click on API Keys
+
+Here you can create and see your API key(s) listed on the API Keys page.  Simply copy and paste this API Key into the Cloudmersive PDF.
+
+Now you are ready to start using the Cloudmersive PDF.
+
+
+## Deployment instructions
+Please use [these instructions](https://docs.microsoft.com/en-us/connectors/custom-connectors/paconn-cli) to deploy this connector as custom connector in Microsoft Flow and PowerApps
```
Comment (dougelliott) [inline connectors/Cloudmersive PDF/readme.md]: @Cloudmersive Could this please be changed to read "...as a custom connector in Microsoft Power Automate and Power Apps". Thanks!

Comment (dougelliott) [inline connectors/Cloudmersive PDF/readme.md]: @Cloudmersive Could this please be changed to read "...as a custom connector in Microsoft Power Automate and Power Apps". Thanks!

File: connectors/Cloudmersive PDF/apiDefinition.swagger.json (hunk 1 above)
Comment (Cloudmersive) [reply connectors/Cloudmersive PDF/apiDefinition.swagger.json]: Fixed

File: connectors/Cloudmersive PDF/readme.md (hunk 2 above)
Comment (Cloudmersive) [reply connectors/Cloudmersive PDF/readme.md]: Fixed

**Comment by Cloudmersive** [comment]
Added requested updates @dougelliott @sriyen-msft

File: connectors/Cloudmersive PDF/readme.md (hunk 2 above)
Comment (dougelliott) [reply connectors/Cloudmersive PDF/readme.md]: Perfect, thank you!

File: connectors/Cloudmersive PDF/apiDefinition.swagger.json (hunk 1 above)
Comment (sriyen-msft) [reply connectors/Cloudmersive PDF/apiDefinition.swagger.json]: @Cloudmersive This swagger looks different and disorganized than the swagger you have submitted in the portal. Please upload the same artifacts here which you have submitted for certification in the portal

File: connectors/Cloudmersive PDF/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,5001 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "v1",
+        "title": "Cloudmersive PDF",
+        "description": "PDF lets you effortlessly create, convert, process, edit, work with, encrypt, and sign PDF files."
+    },
```
Comment (sriyen-msft) [inline connectors/Cloudmersive PDF/apiDefinition.swagger.json]: Please add api annotation back to the the info.
```
,
    "x-ms-api-annotation": {
      "status": "Preview"
    }
```

Comment (Cloudmersive) [reply connectors/Cloudmersive PDF/apiDefinition.swagger.json]: Added x-ms-api-annotation per your request

**Comment by Cloudmersive** [comment]
Made requested updates @sriyen-msft

**Pull request merged**